doc/sphinx/proofs/automatic-tactics/logic.rst

```diff
@@ -192,19 +197,6 @@ some incompatibilities.
       congruence.
       Qed.
 
-.. tacv:: congruence @natural
-
-  Tries to add at most :token:`natural` instances of hypotheses stating quantified equalities
-  to the problem in order to solve it. A bigger value of :token:`natural` does not make
-  success slower, only failure. You might consider adding some lemmas as
-  hypotheses using assert in order for :tacn:`congruence` to use them.
-
-.. tacv:: congruence with {+ @term}
-   :name: congruence with
-
-   Adds :n:`{+ @term}` to the pool of terms used by :tacn:`congruence`. This helps
-   in case you have partially applied constructors in your goal.
-
 .. exn:: I don’t know how to handle dependent equality.
```

Comment:
These two error messages should be indented to appear in the description of `congruence`.

Reply:
Not yet resolved.

Reply:
Got it.